posthog/api/capture.py

```diff
@@ -306,4 +307,5 @@ def capture_internal(event, distinct_id, ip, site_url, now, sent_at, team_id, ev
         sent_at=sent_at,
         event_uuid=event_uuid,
     )
-    log_event(parsed_event, event["event"])
+    partition_key = hashlib.sha256(f"{team_id}:{distinct_id}".encode()).hexdigest()
```

Comment:
Sanity check: Will plugin server be able to hash this identically way if needed in the future?

Reply:
Yup, we have team id and distinct id there and we can get sha256 digests the same way

actually - this reminds me we do produce to kafka from the plugin server already, will make sure we hash there too

Reply:
tagged you in a comment about this